### ISSUE TITLE
AWS region parameter

### ISSUE THREAD

**Issue by silazare** [Nordstrom/kafka-connect-sqs#19]
Hello!
I'm trying to create the connector with AWS Managed Kafka Service (MSK) and facing the issue with region not defined:

```
[Worker-096e720ff778a1351] com.amazonaws.SdkClientException: Unable to find a region via the region provider chain. Must provide an explicit region in the builder or setup environment to supply a region.
```

As far as I understood Region could be passed via Environment only and not via Parameters, as per docker-compose example.

https://github.com/Nordstrom/kafka-connect-sqs/blob/e3140e51f4de47f7a34e26f9892d35c8c75b12e6/src/main/java/com/nordstrom/kafka/connect/sqs/SqsClient.java#L43

https://github.com/Nordstrom/kafka-connect-sqs/blob/master/src/main/java/com/nordstrom/kafka/connect/sqs/SqsConnectorConfigKeys.java#L31

Please suggest if any chance to pass AWS_REGION via parameter, because AWS MSK seems doesn't support env variables ?

**Comment by peterehik-px** [comment]
I created another issue for this with a PR: https://github.com/Nordstrom/kafka-connect-sqs/issues/22. Not sure if I needed to create another issue, can delete it if y'all want me to.

**Comment by dylanmei** [comment]
PR #22 merged and released with [v1.4.0](https://github.com/Nordstrom/kafka-connect-sqs/releases/tag/v1.4.0)
